packages/tempus/src/Tempus_IntegratorForwardSensitivity_impl.hpp

```diff
@@ -326,35 +326,35 @@ createSensitivityModelAndStepper(
 
 /// Non-member constructor
 template<class Scalar>
-Teuchos::RCP<Tempus::IntegratorForwardSensitivity<Scalar> >
+Teuchos::RCP<IntegratorForwardSensitivity<Scalar> >
```

Comment:
So I thought this is OK and it passed tests.  Tests are in the Tempus_Test or Tempus_Unit_Test namespaces, so there are out side the Tempus namespace and I have never used "using namespace Tempus".  Is there a case that this will not work?

Reply:
These objects are inside the Tempus namespace and this code is as well? I'm just puzzled about the original usage and the change; not implying that it won't work.

Reply:
This was cleanup.  Like you mention, these objects and code are inside the Tempus namespace.  I am not sure why the original author (Eric Phipps) did it this way.  I was hopefully removing a possible double namespace issue (i.e., Tempus::Tempus:: ...).